federation-js/src/composition/compose.ts

```diff
@@ -655,5 +649,5 @@ export function composeServices(services: ServiceDefinition[]) {
    * and every field that was extended. Fields that were _not_ extended (added on the base type by the owner),
    * there is no `serviceName`, and we should refer to the type's `serviceName`
    */
-  return { schema: schema as ComposedGraphQLSchema, errors };
+  return { schema, errors };
```

Comment:
A small part of me is curious if this `composeServices` should have symmetry with `composeAndValidate`? (In that it would return `CompositionResult`, but without applying the validations.) Is that even conceivable?

Reply:
Yeah, I like this suggestion quite a bit and nice observation!
d8aeb2a